modules/log6x-code-samples.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// observability/logging/log6x-clf.adoc
+
+
+:_mod-docs-content-type: REFERENCE
+[id="log6x-code-samples"]
+= Code Samples
+
+[source,yaml]
+----
+# Metadata Section: Defines the basic identification of the resource
```

Comment:
Line 12-35  seems chaotic. we needn't explain name, namespace here. suggest use example below.

apiVersion: logging.openshift.io/v1
kind: ClusterLogForwarder
metadata:
  name: instance
spec:
  filters:
  - name: audit-filter
    type: kubeAPIAudit
    kubeAPIAudit:
      omitStages:
      - "RequestReceived"
      rules:
       - level: <level_type1>  annotation xx
       - level: <level_type2>  annotation xx